lib/blocks.php

```diff
@@ -115,6 +115,27 @@ function gutenberg_render_block( $block ) {
 	return '';
 }
 
+/**
+ * Prefixes the core namespace ('core/') to the block type name if the namespace isn't found in the block type name
+ *
+ * @since 2.7.0
+ *
+ * @param  string $block_type Name of the block type.
+ * @return string             Name of the block type, prefixed by the core namespace if needed.
+ */
+function gutenberg_prefix_core_namespace_if_not_found( $block_type ) {
+	$block_type = trim( $block_type );
```

Comment:
We will not expect leading/trailing whitespace from where the function is being called right now (gutenberg_process_block_comment()). 

But this function can also be called as a library function from other functions in future. So, I thought it might be a good idea to trim the input since  a `normalized block type` cannot contain leading/trailing whitespace.

What do you think @aduth ?